libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/resourcelists/ResourceListConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Bosch Software Innovations GmbH, 2018.
+ * Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.sw360.datahandler.resourcelists;
+
+public class ResourceListConstants {
+
+    public static final String COMPONENT_CLASS_NAME = "component";
```

Comment:
We should use these type names in `SW360Constants`

```
    // Proper values of the "type" member to deserialize to CouchDB
    public static final String TYPE_OBLIGATION = "obligation";
    public static final String TYPE_TODO = "todo";
    public static final String TYPE_RISKCATEGORY = "riskCategory";
    public static final String TYPE_RISK = "risk";
    public static final String TYPE_LICENSETYPE = "licenseType";
    public static final String TYPE_LICENSE = "license";
    public static final String TYPE_VENDOR = "vendor";
    public static final String TYPE_USER = "user";
    public static final String TYPE_COMPONENT = "component";
    public static final String TYPE_RELEASE = "release";
    public static final String TYPE_ATTACHMENT = "attachment";
    public static final String TYPE_PROJECT = "project";
    public static final String TYPE_MODERATION = "moderation";
```

Or rename and reuse this `MAP_DOCUMENT_TYPE_STRING` (ThriftEnumUtils)

```
    private static final ImmutableMap<DocumentType, String> MAP_DOCUMENT_TYPE_STRING = ImmutableMap.of(
            DocumentType.COMPONENT, "component" ,
            DocumentType.RELEASE, "release" ,
            DocumentType.PROJECT, "project",
            DocumentType.LICENSE, "license",
            DocumentType.USER, "user"
    );
```